src/alchemlyb/estimators/__init__.py

```diff
@@ -1,3 +1,21 @@
+class EstimatorMixOut():
+    '''This class creates view for the d_delta_f_, delta_f_, states_ for the
+    estimator class to consume.'''
+    _d_delta_f_ = None
+    _delta_f_ = None
+    _states_ = None
```

Comment:
Using the class-level attributes here is a bit tricky: they act as "read" defaults. They will only provide a value of `None` when `self._d_delta_f_` has not been set yet. As soon as `self._d_delta_f_ = value` has been assigned, an instance level attribute is created with `value`, which shadows the class level ones.

It wasn't immediately obvious to me that this would work but I think it does. (I was worried that attributes would be shared between instances but as long as nobody changes the base class attributes themselves it should be fine.)